### ISSUE TITLE
🗣 metricに関するスレッド

### ISSUE THREAD

**Issue by kuto5046** [kuto5046/kaggle-rainforest#15]
## 概要
PyTorchによるLWLRAPの実装
https://www.kaggle.com/c/rfcx-species-audio-detection/discussion/198418

sklearnのLRAPモジュール
https://scikit-learn.org/stable/modules/model_evaluation.html#label-ranking-average-precision

ここを理解して適切なlossを設定する
